spring-aop/src/test/java/org/springframework/aop/support/PointcutsTests.java

```diff
@@ -56,12 +56,7 @@
 	public static Pointcut allTestBeanMethodsPointcut = new StaticMethodMatcherPointcut() {
 		@Override
 		public ClassFilter getClassFilter() {
-			return new ClassFilter() {
-				@Override
-				public boolean matches(Class<?> clazz) {
-					return clazz.equals(TestBean.class);
-				}
-			};
+			return clazz -> clazz.equals(TestBean.class);
```

Comment:
Many developers use "clazz" because "class" is a protected word. But other words like "type" could be better in such cases. How about following?

> return type -> type.equals(TestBean.class);

Reply:
Thank you for your suggestion, I have modified it.